src/modsim2/model/wandb_logger.py

```diff
@@ -0,0 +1,90 @@
+# The code below taken from pytorch_lightning 1.9.3 and adapted
+# to the project's version of 1.8.3.post1 to avoid updating
+# and breaking everything
```

Comment:
as discussed, let's raise an issue to make this temporary

Reply:
link to issue 37

Reply:
Done